core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.ts

```diff
@@ -64,7 +73,29 @@ export class OpportunitiesListItemComponent {
       if (this.opportunity.progressPercentage) {
         this.progressPercentage = (
           this.opportunity.progressPercentage + '%');
-        this.progressBarStyle = { width: this.progressPercentage };
+        const translationSuggestionTypeIndex = 0;
+        if (this.opportunityType === constants.CONTRIBUTION_RIGHT_CATEGORIES[
```

Comment:
Why not compare against `constants.OPPORTUNITY_TYPE_TRANSLATION`?

Reply:
Done